packaging/README.md

```diff
@@ -0,0 +1,33 @@
+# Distribute and Install nmstate
+The packaging and distribution of nmstate is currently implemented through
+PyPI and Docker images.
+
+First prepare and upload to PyPI and then build and upload the docker image.
+ 
+
+## Package and Upload to PyPI
+The following procedures can be reviewed in detail [here](https://packaging.python.org/tutorials/packaging-projects/#uploading-the-distribution-archives).
+```
+cd <project-path>
+python setup.py sdist bdist_wheel
+twine upload dist/*
```

Comment:
What is twine? Maybe explain it and where to get it. Is it the twine package on Fedora?

Reply:
Line 9 points the reader to a link which describe this in more detail, I just wanted to paster here the commands used to build and upload the package... just for convenience.